fkbeta/fk/views.py

```diff
@@ -31,3 +36,14 @@ def user_profile(request):
         profile_form = UserProfileForm(instance=request.user)
         user_form = UserForm(instance=request.user)
     return render(request, "fk/profile.html", {'user_form': user_form, 'profile_form': profile_form})
+
+
+def member_organizations_list(request):
```

Comment:
This part of the patch is a bit confusing, as it is supposed to be moved from fkprofile, but it is only added, the the code in fkprofile is removed in the next commit.  Just mentioning it here to let everyone know what is happening.